index.html

```diff
@@ -846,6 +846,12 @@
           represented by a <a>Promise</a> that resolves with the
           <a>presentation controllers monitor</a>.
         </p>
+        <p>
+          In a <a>controlling browsing context</a>, the <dfn>default
+          presentation request</dfn>, which is initially set to
+          <code>null</code>, represents the request to use when the user wishes
+          to initiate a <a>presentation connection</a> from the browser chrome.
```

Comment:
Nit: The term "browser chrome" seems to imply a user gesture on the browser window itself.  (Is this term defined somewhere?)

I propose just "the browser", or a phrase that encompasses actions other than clicking a button.  We explicitly mention cases like that in a NOTE discussing granting permission for use of a presentation display.

> For example, the browser chrome could allow the user to pick a display from a menu, or allow the user to tap on an Near Field Communications (NFC) enabled display.